.github/workflows/integration-tests-ignored.yml

```diff
@@ -0,0 +1,25 @@
+# This is a dummy file so when changes occur that do not need smoke tests ran we can still have the check be required
```

Comment:
Nit: can we call this `integration-tests-ignore.yml` instead? This matches the other files that just end in `-ignore.yml`